asmrun/signals_osdep.h

```diff
@@ -154,6 +154,26 @@
  #define CONTEXT_YOUNG_PTR (context->uc_mcontext.gregs[REG_R15])
  #define CONTEXT_FAULTING_ADDRESS ((char *) info->si_addr)
 
+/****************** PowerPC, NetBSD */
```

Comment:
The blocks in this file seem to be ordered by architecture first, OS second, so I think it would make more sense to have this blob in the PowerPC part below. Currently the "PowerPC, BSD" check is clearly designed to also catch NetBSD systems (except it doesn't as the NetBSD configure uses `system=elf`), so it would make sense to have the new code close by, and also maybe to disable the `SYS_netbsd` conditional there?

Reply:
Ok, I'll look at moving this block as appropriate.  The use of `system=elf` doesn't actually work, because "power,elf" seems to imply Linux, as configure then wants to do `"partialld="ld -r -m elf32ppclinux"`.  I have a new change lined up which will instead use `system=netbsd` also for NetBSD/powerpc, I'm just waiting for all the self-tests to complete.